llvm/test/CodeGen/SPIRV/transcoding/atomic_dec.ll

```diff
@@ -0,0 +1,16 @@
+; RUN: llc -O0 -mtriple=spirv64-unknown-unknown %s -o - | FileCheck %s
```

Comment:
Let's add `-verify-machineinstrs` for each new test?

Reply:
Yes, forgot about this, thanks for reminding!